suitcase/tiff_series/tests/conftest.py

```diff
@@ -1,4 +1,5 @@
 from bluesky.tests.conftest import RE  # noqa
 from ophyd.tests.conftest import hw  # noqa
 from suitcase.utils.tests.conftest import (  # noqa
-    example_data, generate_data, plan_type, detector_list, event_type)
+    example_data, generate_data, plan_type, detector_list, event_type,
+    directory_list)
```

Comment:
Why isn't `file_prefix_list` imported/used in this case?

Reply:
@mrakitin because suitcase.tiff_series has a different API for the formatting (`{start[uid]}' as opposed to '{uid}') and so it requires a custom setup.

Reply:
I see. Thanks.